From 9d90ca09312257da6a3ba97e146160e1c68702fc Mon Sep 17 00:00:00 2001
From: Zumeng Chen <zumeng.chen@windriver.com>
Date: Fri, 12 Oct 2018 12:55:20 +0800
Subject: [PATCH 43/51] soc: ti: wkup_m3_ipc: fix compiling error for rproc

commit 1b20bb21a049fc5c4735639e64fe7afa3e8e727d from
https://git.ti.com/cgit/processor-sdk/processor-sdk-linux/?h=processor-sdk-linux-01.00.00

drivers/soc/ti/wkup_m3_ipc.c: In function 'wkup_m3_copy_aux_data':
drivers/soc/ti/wkup_m3_ipc.c:112:18: error: too few arguments to function 'rproc_da_to_va'
  aux_data_addr = rproc_da_to_va(m3_ipc->rproc,
                  ^~~~~~~~~~~~~~

Signed-off-by: Zumeng Chen <zumeng.chen@windriver.com>
Signed-off-by: Xiaolei Wang <xiaolei.wang@windriver.com>
---
 drivers/soc/ti/wkup_m3_ipc.c | 5 ++---
 1 file changed, 2 insertions(+), 3 deletions(-)

diff --git a/drivers/soc/ti/wkup_m3_ipc.c b/drivers/soc/ti/wkup_m3_ipc.c
index 49f75d9c3a8c..df885d5d336c 100644
--- a/drivers/soc/ti/wkup_m3_ipc.c
+++ b/drivers/soc/ti/wkup_m3_ipc.c
@@ -101,9 +101,8 @@ static unsigned long wkup_m3_copy_aux_data(struct wkup_m3_ipc *m3_ipc,
 	void *aux_data_addr;
 
 	aux_data_dev_addr = WKUP_M3_DMEM_START + WKUP_M3_AUXDATA_OFFSET;
-	aux_data_addr = rproc_da_to_va(m3_ipc->rproc,
-				       aux_data_dev_addr,
-				       WKUP_M3_AUXDATA_SIZE);
+	aux_data_addr = rproc_da_to_va(m3_ipc->rproc, aux_data_dev_addr,
+				WKUP_M3_AUXDATA_SIZE, RPROC_FLAGS_NONE);
 	memcpy(aux_data_addr, data, sz);
 
 	return WKUP_M3_AUXDATA_OFFSET;
-- 
2.17.1

